{"links": {"self": {"href": "data/repositories/osrf/drcsim/issues/323/comments/4770582.json"}, "html": {"href": "#!/osrf/drcsim/issues/323#comment-4770582"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/drcsim/issues/323.json"}}, "type": "issue", "id": 323, "repository": {"links": {"self": {"href": "data/repositories/osrf/drcsim.json"}, "html": {"href": "#!/osrf/drcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{247b5408-ae91-4e3c-b2f9-a21994658d11}ts=c_plus_plus"}}, "type": "repository", "name": "drcsim", "full_name": "osrf/drcsim", "uuid": "{247b5408-ae91-4e3c-b2f9-a21994658d11}"}, "title": "Gazebo exiting on simulation machine in the cloud"}, "content": {"raw": "We were running pinned with the robot turning the steering and pressing\ngas.\n\nUnfortunately the logs are on the simulation machine on the cloud (to which\nI don't think we have access to any more).\n", "markup": "markdown", "html": "<p>We were running pinned with the robot turning the steering and pressing\ngas.</p>\n<p>Unfortunately the logs are on the simulation machine on the cloud (to which\nI don't think we have access to any more).</p>", "type": "rendered"}, "created_on": "2013-06-10T18:30:18.041816+00:00", "user": {"display_name": "sachih", "uuid": "{5b4b74fa-1172-4c6f-a38b-54ffaf7dbe2e}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5b4b74fa-1172-4c6f-a38b-54ffaf7dbe2e%7D"}, "html": {"href": "https://bitbucket.org/%7B5b4b74fa-1172-4c6f-a38b-54ffaf7dbe2e%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/8bdeccc7f3489a8a046ba5c80a81892ed=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsS-5.png"}}, "nickname": "sachih", "type": "user", "account_id": "557058:cf468912-43ca-47b2-b8d6-3b90908dc837"}, "updated_on": null, "type": "issue_comment", "id": 4770582}